test/support/test_utils_grpc.ex

```diff
@@ -75,8 +77,90 @@ defmodule Logflare.TestUtilsGrpc do
     boolean = random_key_value(:boolean)
     integer = random_key_value(:integer)
     double = random_key_value(:double)
+    array_of_strings = random_key_value(:array_of_strings)
+    array_of_numbers = random_key_value(:array_of_numbers)
+    array_of_booleans = random_key_value(:array_of_booleans)
 
-    [string, boolean, integer, double]
+    [
+      string,
+      boolean,
+      integer,
+      double,
+      array_of_strings,
+      array_of_numbers,
+      array_of_booleans
+    ]
+  end
+
+  # this is not supported by traces/metrics, only logs
+  # https://github.com/open-telemetry/opentelemetry-specification/pull/2888
+  # https://github.com/open-telemetry/opentelemetry-specification/issues/376
+  defp random_key_value(:array_of_kv) do
```

Comment:
This is not used anywhere. Perhaps expected, I am just noting to double-check :)

Reply:
yea i intend to use it again when we support logs 😄